doc/release-notes/8424-signposting.md

```diff
@@ -0,0 +1,42 @@
+# Signposting for DV 5.9
+
+This branch adds [Signposting](https://signposting.org/) support to DV 5.9.
+
+There are 2 Signposting profile levels, level 1 and level 2. In this implementation, 
+ * level 1 links are shown in 
+HTTP header, which can be fetched by `curl -I https://domain/link-to-article`. 
+ * The level 2 linkset can be fetched by visiting the dedicated linkset page for 
+   that artifact. The link can be seen in level 1 links with key name `rel="linkset"`.
+
+The configuration is stored as JSON string in the `Bundle.properties` file, key name is
+`signposting.configuration.SignpostingConf`. Please see a sample configuration below with explaination for each of the
+config items.
+
+```json
+{
+  "license": {
+    "CC0": "https://creativecommons.org/licenses/cc0/"
+  },
+  "describedby": {
+    "doi": "https://doi.org/",
+    "type": "application/vnd.citationstyles.csl+json"
+  },
+  "useDefaultFileType": true,
+  "defaultFileTypeValue": "https://schema.org/Dataset",
```

Comment:
starting with `userDefaultFileType`, these are all Signposting specific configuration. 
Signposting can show different FileType according to the given metadata, however, in dataverse these metadata is not there yet. I left this to be configurable. If `true`, the type will be the set type, else it will be the actual type given in metadata.